### PR TITLE
In CMAKE use VERSION of project command

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

**Review by ohm314 — APPROVED** [review]
lgtm - why is this better? just shorter?

**Comment by alkino** [comment]
It's shorter and more straightforward. Variables are declared automatically.

**Comment by alkino** [comment]
Please retest

**Comment by pramodk** [comment]
@alkino 's `make` should be aliased to `make && make test && make cmake-format  && make clane-format` :smiley:

**Comment by alkino** [comment]
> make && make test && make cmake-format && make clang-format

`make -j8 all test cmake-format clang-format`

**Comment by pramodk** [comment]
even better! thanks!

File: CMakeLists.txt [hunk 1]
```diff
@@ -6,13 +6,14 @@
 # =============================================================================
 
 cmake_minimum_required(VERSION 3.3.0 FATAL_ERROR)
-project(NMODL CXX)
+project(
+  NMODL
+  VERSION 0.2
+  LANGUAGES CXX)
```
Comment (pramodk) [inline CMakeLists.txt:12]: just a nitpick : does cmake-format force splitting this on multiple line? quite superfluous !

Comment (alkino) [reply CMakeLists.txt:12]: Yes, I agree ;)

Comment (pramodk) [reply CMakeLists.txt:12]: @alkino : note there is `# ~~~` marker in cmake-format to disable it (and avoid absurd formatting)

Comment (alkino) [reply CMakeLists.txt:12]: This is the default rule of no more than 3 words in a command on the same line. Maybe we can change it to 10?

Comment (pramodk) [reply CMakeLists.txt:12]: do you mean number of args settings? I don't know if there is an option but I would say why not!

**Pull request merged**